components/mpas-ocean/src/shared/mpas_ocn_surface_land_ice_fluxes.F

```diff
@@ -677,14 +678,24 @@ subroutine ocn_surface_land_ice_fluxes_init(err)!{{{
 
       select case (trim(config_land_ice_flux_mode))
       case ('standalone','Standalone','STANDALONE')
-         landIceFluxesOff = .false.
-         standaloneOff    = .false.
+         landIcePressureOn = .true.
+         landIceFluxesOn = .true.
+         landIceStandaloneOn = .true.
```

Comment:
@xylar thanks for the suggestion. I think this is much improved, we should never not use positive logic ;).  I also made the logic flags public if they are used elsewhere (specifically diagnostics). 

I thought about substituting `landIceCoupledOn` for `standAloneOff` but that doesn't make sense for these last two cases, i.e. the new `LandIceStandAloneOn` is really what is needed to run the subroutine `ocn_surface_land_ice_fluxes_build_arrays`.

See also lines just below this for the different cases.

Reply:
I actually introduced a lot of the internal "off" flags because we were always only checking whether something was off and returning, so the only logical check in the code was for an "off" case and we kept negating an "on" flag for that conditional. So if there are more "on" checks now for the allocates, it makes sense to switch to an "on" flag.

Reply:
Thanks @mark-petersen.  @philipwjones, I think even if the flags happened to fit the places in the code where logicals were needed, the combination of `*Off` and `*On` logicals were confusing.  But it's probably my mistake not reviewing the PR(s) that introduced the changes more thoroughly in the first place.